qa-heal.planx-pla.net/metadata/aggregate_config.json

```diff
@@ -1,1240 +1,1296 @@
 {
-    "gen3_commons": {
-        "HEAL": {
-            "mds_url": "https://qa-heal.planx-pla.net/",
-            "commons_url": "https://qa-heal.planx-pla.net/"
+  "configuration": {
+    "schema": {
+      "__manifest": {
+        "type": "array",
+        "properties": {
+          "file_name": {
+            "type": "string"
+          },
+          "file_size": {
+            "type": "integer"
+          }
+        }
+      },
+      "commons_url": {},
+      "tags": {
+        "type": "array"
+      },
+      "advSearchFilters": {
+        "type": "array",
+        "default": []
+      },
+      "study_description_summary": {},
+      "project_title": {}
+    },
+    "settings": {
+      "cache_drs": true
+    }
+  },
+  "adapter_commons": {
+    "HEAL": {
+      "mds_url": "https://qa-heal.planx-pla.net/",
+      "commons_url": "https://qa-heal.planx-pla.net/",
+      "adapter": "gen3",
+      "config" : {
+        "guid_type": "unregistered_discovery_metadata",
```

Comment:
is this "unregistered_discovery_metadata" only for unregistered HEAL studies? how about registered heal studies? I think that is discovery_metadata? how are those loaded in the aggMDS if guid_type is only refined for unregistered_discovery_metadata here?